virttest/utils_test/__init__.py

```diff
@@ -1087,8 +1087,10 @@ def runtest(self):
                 avocado_cmd += " --vt-no-filter \"%s\"" % self.vt_no_filter
             if self.vt_arch:
                 avocado_cmd += " --vt-arch %s" % self.vt_arch
+            if self.guest_image:
+                avocado_cmd += " --vt-guest-os %s" % self.guest_image
             if self.vt_extra_params:
-                avocado_cmd += " --vt-extra-params \"%s\"" % self.vt_extra_params
+                avocado_cmd += " --vt-extra-params %s" % self.vt_extra_params
```

Comment:
any reason for this change, this was there to handle any space/spl chars to be wrapped right, what changed now?

Reply:
for extra params, we really don't need to wrap, and framework errors out if we try to wrap it. where we give `key='value'`